ledger/src/leader_schedule.rs

```diff
@@ -48,6 +64,28 @@ impl LeaderSchedule {
     pub fn num_slots(&self) -> usize {
         self.slot_leaders.len()
     }
+
+    /// Returns indices starting from 'offset' where pubkey is the leader.
```

Comment:
nit: how about defining `offset`, something like

"Given an input 'offset', an index into the leader schedule, the function returns an iterator of indices  `i > offset` at which the given pubkey is the leader`

Reply:
Expanded the comment.